### DISCUSSION TITLE
Use journald for pod logs

### DISCUSSION THREAD

**Discussion by mawiegand** [k3s-io/k3s#9961] [Q&A] [answered]
Hey, I would like to use Journald for the pod logs. Is this somehow possible e.g. via a configuration option or something else?

Any help is welcome! :)

**Comment by brandond** [comment, marked as answer]
You would have to get this implemented in Kubernetes itself. The kubelet is currently hardcoded to place pod logs in text format at /var/log/pods, and expects them to be there in files with specific names.
